pkg/ui/workspaces/db-console/src/views/cluster/containers/nodeGraphs/dashboards/replication.tsx

```diff
@@ -102,6 +104,24 @@ export default function (props: GraphDashboardProps) {
       </Axis>
     </LineGraph>,
 
+    <LineGraph
+      title="Average Replica CPU per Node"
+      tooltip={`Moving average of all replica CPU usage on each node per second.
+         Tracks roughly the last 30 minutes of usage. Used for load-based
+         rebalancing decisions.`}
+    >
+      <Axis units={AxisUnits.Duration} label="CPU time">
+        {_.map(nodeIDs, nid => (
```

Comment:
nit: prefer native array map over lodash's map

Reply:
Updated to use native map. I also pushed a preceding commit that replaces `_.map` with the native array map for the other graphs in `replication.tsx`.